### PR TITLE
DDF-3210 Updated catalog taxonomy to support new DCMI-defined types

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Updates the DDF DataTypes to conform to the DCMI types.
#### Who is reviewing it? 
@emmberk @brendan-hofmann 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@figliold
#### How should this be tested? (List steps with links to updated documentation)
Install ddf and ingest a pdf and video file through Intrigue. Verify that they have the updated datatypes of Text and Moving Image.
#### What are the relevant tickets?
[DDF-3210](https://codice.atlassian.net/browse/DDF-3210)
#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

File: distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc [hunk 1]
```diff
@@ -84,7 +84,7 @@ __Coordinates must be in *lon-lat* coordinate order__
 |resource-size
 |Size in bytes of resource.
 |String
-|Although this type cannot be changed for legacy reasons, its value should always be a parseable whole number.
+|Although this type cannot be changed for legacy reasons, its value should always be a parseable whole number
```
Comment (figliold) [inline distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc]: ✏️  Shouldn't parseable be parsable?

File: distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc [hunk 2]
```diff
@@ -143,10 +143,10 @@ __Coordinates must be in *lon-lat* coordinate order__
 |
 
 |datatype
-|The generic type(s) of the resource. http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements#terms-type[Dublin Core terms-type] and extended to include other DDF supported types.
+|The generic type(s) of the resource including the http://dublincore.org/documents/dcmi-type-vocabulary/[Dublin Core terms-type]. DCMI Type term labels are expected here, as opposed to term names.
```
Comment (figliold) [inline distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc]: ❓ Do you really need this comma here: "...expected here, as opposed to term..."?  I think you can remove it.

File: distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc (hunk 1 above)
Comment (clockard) [reply distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc]: Fixed

File: distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc (hunk 2 above)
Comment (clockard) [reply distribution/docs/src/main/resources/_contents/_data/core-attributes-table-contents.adoc]: Fixed

**Comment by clockard** [comment]
build now

**Pull request merged**